test/e2e/activator_test.go

```diff
@@ -124,3 +128,105 @@ func TestActivatorOverload(t *testing.T) {
 	}
 	group.Wait()
 }
+
+func TestActivatorRevisionTimeout(t *testing.T) {
```

Comment:
I'd move it to the existing timeout test and basically what you want is
```
tests := []struct{
name string
tbc string
}{ "activator", "-1"},{"direct", "0"}}

Then create the revision (you don't even need 2 services, just 2 revisions, but I don't care much about that) with given TBC settings.
Rest of the flow should be the same.

Reply:
Is that okay even though those are conformance tests and the TBC annotations are not part of conformance? :thinking:

Reply:
I think it's fine, since it should be ignored by other systems, which basically ends executing the same test twice. Not ideal, but I think better than having 2 big tests to maintain and support and remember to update.